lib/myxql/protocol/auth.ex

```diff
@@ -1,7 +1,7 @@
 defmodule MyXQL.Protocol.Auth do
   @moduledoc false
 
-  # TODO: remove when we require Elixir v1.10+
+  # TODO: remove when we require Elixir v1.14+
```

Comment:
the comment is wrong, can you remove it? It is fine that we call `require Bitwise; Bitwise.bxor`.

Reply:
It is fine but we can remove `require Bitwise` hereon Elixir 1.14 as it's  [deprecated](https://github.com/elixir-lang/elixir/blob/v1.14/CHANGELOG.md) and this file does not use operator (which requires `import Bitwise` like other modules)

But Elixir <1.14 needs `require` here.

Reply:
sorry, the comment was correct after all. The change between v1.9 and v1.10 is basically:

```diff
-  defmacro bxor(left, right) do
+  def bxor(left, right) do
```

that's why we previously needed a require and nowadays we don't.